bool            get_param
---
localization_param    parameters
